grammars/html.cson

```diff
@@ -335,6 +335,26 @@
     'match': '<>'
     'name': 'invalid.illegal.incomplete.html'
   }
+  {
+    'begin': '(?<=>).{1}'
```

Comment:
@kevinsawicki in the case that these are empty, the editor will freeze, i.e. `><` and `<` would freeze the editor. So I've added `.{1}` to each of the begin patterns. Strangely just a `.` or a `[.]` doesn't work. 

In practice, snippets and things are shown in the correct places, and I dont think i'm stepping on any other rules.

Do you have a better way to handle this?